### PR TITLE
load require.js & direct to python-env repo

### PR DESCRIPTION
**Pull request by tannavee**
index.html loads require.js, and activateThebelab.js points to our private python repo

### CODE REVIEW

File: src/scripts/activateThebelab.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ const getLanguage = () => {
 const getConfig = (language) => {
   const config = {
     binderOptions: {
-      repo: 'binder-examples/requirements',
+      repo: 'LibreTexts/ckeditor-binder-plugin',
+      ref: 'Python-Env'
```
Comment (narrator0) [inline src/scripts/activateThebelab.js]: you need a comma at the end.

**Review by narrator0 — CHANGES_REQUESTED** [review]

**Comment by tannavee** [comment]
somehow it is saying that there is an issue with index.js, plugin.js, and registerPlugin.js

File: src/scripts/activateThebelab.js (hunk 1 above)
Comment (tannavee) [reply src/scripts/activateThebelab.js]: thank you

**Comment by narrator0** [comment]
It's because of the missing comma. I think just fix that and everything will be fine.

**Review by narrator0 — APPROVED** [review]

**Pull request merged**